ParallelMatrixMultiplication/ParallelMatrixMultiplication/MatrixUserInterface.cs

```diff
@@ -0,0 +1,121 @@
+        /// Performs performance testing of sequential and parallel matrix multiplication
+        /// for various matrix sizes with a set number of runs.
+        /// </summary>
+        /// <param name="n">The number of runs for each test.</param>
+        public static void UsersMatrixTests(int n)
+        {
+            var testCases = new List<(int RowsA, int ColumnsA, int RowsB, int ColumnsB)>()
+            {
+                (100, 100, 100, 100),
+                (200, 200, 200, 200),
+                (300, 300, 300, 300),
+                (400, 400, 400, 400),
+                (500, 500, 500, 500),
+            };
+
+            Console.WriteLine($"Проведение тестирования с {n} запусками");
+            Console.WriteLine("\nРазмеры матриц | " +
+                "Матожидание(последовательное) | " +
+                "СКО(последовательное) | " +
+                "Матожидание(параллельное) | " +
+                "СКО(параллельное) \n");
+
+            foreach (var testCase in testCases)
+            {
+                int rowsA = testCase.RowsA;
+                int columnsA = testCase.ColumnsA;
+                int rowsB = testCase.RowsB;
+                int columnsB = testCase.ColumnsB;
+
+                Matrix matrixA = Matrix.GenerateRandomMatrix(rowsA, columnsA);
+                Matrix matrixB = Matrix.GenerateRandomMatrix(rowsB, columnsB);
+
+                var sequentialTimes = new List<double>();
+                var parallelTimes = new List<double>();
+
+                int numOfThreads = Environment.ProcessorCount;
+
+                for (int i = 0; i < n; ++i)
+                {
+                    Stopwatch stopwatch = Stopwatch.StartNew();
+                    MatrixMultiplier.SequentialMultiplication(matrixA, matrixB);
+                    stopwatch.Stop();
+                    sequentialTimes.Add(stopwatch.Elapsed.TotalMilliseconds);
+
+                    stopwatch.Restart();
+                    MatrixMultiplier.ParallelMultiplication(matrixA, matrixB, numOfThreads);
+                    stopwatch.Stop();
+                    parallelTimes.Add(stopwatch.Elapsed.TotalMilliseconds);
+                }
+
+                double sequentialExpectation = CalculateMathematicalExpectation(sequentialTimes);
+                double parallelExpectation = CalculateMathematicalExpectation(parallelTimes);
+
+                double sequentialStandardDeviation = CalculateStandardDeviation(sequentialTimes, sequentialExpectation);
+                double parallelStandardDeviation = CalculateStandardDeviation(parallelTimes, parallelExpectation);
+
+                Console.WriteLine($"{rowsA}x{columnsA} * {rowsB}x{columnsB} | {sequentialExpectation:F2} мс | {sequentialStandardDeviation:F2} | {parallelExpectation:F2} | {parallelStandardDeviation:F2}\n");
+            }
+        }
+
+        private static double CalculateMathematicalExpectation(List<double> values)
+        {
+            return values.Sum() / values.Count;
+        }
```

Comment:
для однострочных функций лучше использовать синтаксис лямбд
C# позволяет, а вот Java такого сделать не даст :)
```suggestion
        private static double CalculateMathematicalExpectation(List<double> values) => values.Sum() / values.Count;
```